src/Microsoft.SqlTools.ManagedBatchParser/Properties/AssemblyInfo.cs

```diff
@@ -15,7 +14,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Microsoft.SqlTools.ManagedBatchParser")]
-[assembly: AssemblyCopyright("Copyright ©  2019")]
+[assembly: AssemblyCopyright("Copyright © 2022")]
```

Comment:
all these attributes should be msbuild properties instead

Reply:
AssemblyInfo.cs file should probably not even exist

Reply:
We should remove the copyright year too, we never use it anywhere else. `Copyright (c) Microsoft Corporation`